pkg/metrics/metrics.go

```diff
@@ -0,0 +1,53 @@
+package metrics
+
+import (
+	"net"
+	"net/http"
+
+	c "github.com/scality/cosi-driver/pkg/constants"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"k8s.io/klog/v2"
+)
+
+var (
+	RequestsTotal = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "cosi_requests_total",
```

Comment:
maybe we could prefix `s3_`  most of our S3C metrics have this prefix to differentiate easily from the RING (expect utapi & scuba)

Reply:
This service is not a part of RING but would be deployed on customer's kubernetes clusters. Its deployed externally.

Per https://prometheus.io/docs/practices/naming/ it should be component <component>_<operation>_<type>.
As this is not an s3 metric but a COSI metric, it makes sense to keep cosi or even better `cosi_driver` as prefix
I would change this to `cosi_driver`

Reply:
Oh Ok, so we will not scrape thing in our prometheus right and will not implement dashboards ? Or maybe we could implement dashboard that use an external datasource, or that they could copy in their own system